docs/website/docs/developers/debugging/gpu.md

```diff
@@ -0,0 +1,177 @@
+---
+hide:
+  - tags
+tags:
+  - GPU
+  - CUDA
+  - Metal
+  - ROCm
+  - Vulkan
+icon: material/chip
+---
+
+# GPU debugging playbook
```

Comment:
After seeing the rendered page, I think it would also be helpful to break up the text with some images, diagrams, code blocks, or even admonitions/call-outs (https://squidfunk.github.io/mkdocs-material/reference/admonitions/) to add some color.

For example,

* scroll through https://iree.dev/developers/performance/benchmarking/ to see how code blocks with syntax highlighting break up the paragraphs
* https://iree.dev/guides/ml-frameworks/pytorch/ has a few diagrams
* https://iree.dev/building-from-source/getting-started/ has lots of admonitions/call-outs

Reply:
(fine to merge without this, just something to keep in mind while authoring or refreshing docs)

Reply:
SG! Made it more colorful.